development/acpica/acpica.SlackBuild

```diff
@@ -74,9 +70,7 @@ find -L . \
  -o -perm 440 -o -perm 400 \) -exec chmod 644 {} \;
 
 cd generate/unix
-  CFLAGS="$SLKCFLAGS -fcommon -Wno-error=format-overflow -Wno-error=stringop-truncation" \
-  CXXFLAGS="$SLKCFLAGS" \
-  make BITS=$BITS
+  make OPT_CFLAGS="$SLKCFLAGS" \
```

Comment:
there's a stray "\\" at the end, but perhaps better to keep the OPT_CFLAGS on a separate line

Reply:
Fixed, thank you